src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from "react-router-dom";
+
+import { footerData } from "../../constans.ts";
+import { themes } from "../../themes";
+import { Typography } from "../typography/Typography";
+import * as S from "./Footer.styled.ts";
+
+export const Footer = () => {
+	return (
+		<S.FooterWrapContainer>
+			<Typography
+				variant="h6"
+				color={themes.light.background}
+			>
+				Fragen? Einfach anrufen:123-987-345
+			</Typography>
+			<S.FooterWrapContainer>
+				{footerData.map((item) => (
+					<S.FooterItemWrap key={item}>
+						<Typography
+							variant="body1"
+							color={themes.light.background}
+						>
+							<Link to={`/${item}`}>{item}</Link>
```

Comment:
Linki jeżeli masz gdzieś już zapisane to powinny posiadać tego slasha w definicji, wtedy tylko zrobimy `to={item.href}`